DTS-STN/Alpha-Site
# Privacy page redesign

# Description

[67714 - DEV: Add "On This Page" section and Information Alert component to Privacy Policy page](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities/?workitem=67714)

This PR adds the `TableOfContents` component, which accepts an array of headings with their IDs and displays them in a list. I've also updated the privacy page content fragment and converted the page to use the data from AEM. I've also added the Alert component to the page as per the new design.


## Test Instructions

1. Pull in branch
2. Type `yarn dev`
3. Navigate to `/signup/privacy`
4. Ensure the page looks like the new design and the table of contents functions properly

## Checklist


## Product and Sprint Backlog

[ADO](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities)
